Structure_Engine/Query/Normal.cs

```diff
@@ -34,29 +36,12 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Returns the bars local Z-axis")]
+        [Input("bar","The relevant Bar")]
```

Comment:
Space needed after comma.
What is the 'relevant' bar? Maybe a better description for the input is needed?